tests/e2e/vLLM/recipes/INT8/recipe_w8a8_dynamic_asym.yaml

```
@@ -0,0 +1,17 @@
+quant_stage:
+  quant_modifiers:
+    SmoothQuantModifier:
+      smoothing_strength: 0.8
+      mappings:
+      - - ['re:.*q_proj', 're:.*k_proj', 're:.*v_proj']
+        - re:.*input_layernorm
+      - - ['re:.*gate_proj', 're:.*up_proj']
+        - re:.*post_attention_layernorm
+    GPTQModifier:
+      sequential_update: false
+      ignore: [lm_head]
+      config_groups:
+        group_0:
+          weights: {num_bits: 8, type: int, symmetric: true, strategy: channel}
```

i think we want asym here?
```suggestion
          weights: {num_bits: 8, type: int, symmetric: false, strategy: channel}
```

---

vLLM doesn't support asym weights so we can keep it as symmetric.
see here: https://github.com/vllm-project/vllm/blob/258bf621d5e533b01026b73fbfb31b746f68684f/vllm/model_executor/layers/quantization/compressed_tensors/schemes/compressed_tensors_w8a8_int8.py#L96